config/Dockerfiles/Jenkinsfile

```diff
@@ -19,7 +19,7 @@ DOCKER_REPO_URL = env.DOCKER_REPO_URL ?: '172.30.254.79:5000'
 // if it's not defined you will get all the possible providers
 // listed here.
 
-LPROVIDERS = env.LINCHPIN_PROVIDERS ?: 'dummy duffy libvirt aws'
```

Comment:
why do we have to disable libvirt?

Reply:
@p3ck Currently, we have a race condition where the libvirt provisioning happens for each distro in a single pod, sharing a single network. As we've discovered, there will be a race condition where one of the three distros will get the libvirt default network and the other two will then be unable to get IP addresses and timeout eventually. See #634 for more detail.